docs/pages/xy-grid.md

```diff
@@ -26,6 +26,8 @@ The XY grid is supported in Chrome, Firefox, Safari 6+, IE10+, iOS 7+, and Andro
 
 **XY grid is the default Foundation grid**. It is present by default in `foundation.css` of CDN link or package managers. In Sass, it will be generated by default by `@include foundation-everything` (unless `$global-flex` or `$xy-grid` are set to false).
 
+If `$xy-grid: false` is set, the flex grid is used instead.
```

Comment:
I think we should rather reformulate the whole sentence. Like:

> ... (unless `$xy-grid` or `$global-flex` is set to false, it will then generate the Flex Grid or the Float Grid depending on `$global-flex`).

Or

> ... (unless `$xy-grid` or `$global-flex` is set to false, it will then generate the Flex Grid if `$global-flex` is true, or the Float Grid otherwise).

Reply:
I'll think about it and push a new commit tomorrow. Thanks for the feedback.